scania-digital-design-system/sdds
# Bug/eslint fixes

**Describe pull-request**  
_Describe what the pull-request is about_
Adding back ESlint in the project to make easier setting up new rules from linting and prettier 

**Solving issue**  
_Add which issue this pull-request solves by adding # plus the number of the issue (for example #123)_
Fixes: #180 [AB#961](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/961) 

**How to test**  
_Add description how to test if possible_
1. Create a seperate branch from this branch
2.  Try and change some files and do a test commit, it should run "npm run lint" command 

**Screenshots**  
_If applicable, add screenshots to help explain_

**Additional context**  
_Add any other context about the pull-request here._
